simulators/swan/v41.31/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM ubuntu:22.04 as test_env
+
+RUN apt-get update \
+    && apt-get install -y \
+    wget \
+    unzip &&\
+    wget https://storage.googleapis.com/inductiva-api-demo-files/swan-input-example.zip -P /home/ && \
```

Comment:
Can we start getting the examples in the code repo?
Or are these generated from the code repo?

Reply:
This and the code repo are not related. The code repo came later due to the fact that some simulators do not store a history of older versions.

Reply:
But this is an example for testing purposes, right?
It would make sense to make this Kutu repo self-contained and not depend on an artifact that lives elsewhere, that can be moved, etc... The code for testing Kutu should move to the Kutu repo. 
Or, we have a repo just for holding the tests and example code, from which we draw all the tests.
Please bring this issue to the attention of @hpenedones .